channels/ar.m3u

```diff
@@ -25,8 +25,6 @@ http://panel.dattalive.com:1935/8204/8204/playlist.m3u8
 https://59c5c86e10038.streamlock.net/8204/8204/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal11LaRioja.ar" tvg-name="Canal 11 La Rioja" tvg-country="AR" tvg-language="" tvg-logo="http://www.fenix951.com.ar/nuevo_2013/a2017/assets/imgapp/logocanal.jpeg" group-title="",Canal 11 La Rioja
 http://stmv4.questreaming.com/fenixlarioja/fenixlarioja/playlist.m3u8
-#EXTINF:-1 tvg-id="Canal12PuertoMadryn.ar" tvg-name="Canal 12 Puerto Madryn" tvg-country="AR" tvg-language="" tvg-logo="" group-title="",Canal 12 Puerto Madryn
-https://59db7e671a1ad.streamlock.net/madryntv/madryntv/playlist.m3u8
```

Comment:
Change domain name for 5f700d5b2c46f.streamlock.net to prevent any certificate issue.

https://5f700d5b2c46f.streamlock.net/madryntv/madryntv/playlist.m3u8

Not 24/7.